x/crosschain/keeper/msg_server_whitelist_erc20.go

```diff
@@ -35,6 +36,10 @@ func (k Keeper) WhitelistERC20(goCtx context.Context, msg *types.MsgWhitelistERC
 			return nil, sdkerrors.Wrapf(types.ErrInvalidAddress, "ERC20 contract address (%s) already whitelisted on chain (%d)", msg.Erc20Address, msg.ChainId)
 		}
 	}
+	tss, found := k.GetTSS(ctx)
+	if !found {
+		return nil, errorsmod.Wrapf(types.ErrCannotFindTSSKeys, "Cannot create new admin cmd")
```

Comment:
> `Cannot create new admin cmd`

Is it the right error message? It seems not related to the error to me

Reply:
whitelistERC20 is an action of type admin cmd, we will have more of these in the future .
Added some details to the error print so that it makes more sense.